### PR TITLE
feat: fruit fly validation for organism_tissue_ontology_term_id

### PR DESCRIPTION
**Pull request by joyceyan**
## Reason for Change

https://app.zenhub.com/workspaces/cs-platform-661fffc5c6339904f8f42cb2/issues/gh/chanzuckerberg/single-cell-curation/1047

## Changes

- adds fruit fly validation rules for `organism_tissue_ontology_term_id`

## Testing

added unit testing

## Notes for Reviewer

### CODE REVIEW

File: cellxgene_schema_cli/cellxgene_schema/schema_definitions/schema_definition.yaml [hunk 1]
```diff
@@ -233,6 +233,27 @@ components:
...
+              'organism_tissue_ontology_term_id' MUST be the most accurate descendant
+              of FBbt:10000000 for fruit fly anatomical entity and MUST NOT be FBbt:00007002
+              for cell or any of its descendants.
+            type: curie
+            curie_constraints:
+              ontologies:
+                - FBbt
+              allowed:
+                ancestors:
+                  ZFA:
```
Comment (nayib-jose-gloria) [inline cellxgene_schema_cli/cellxgene_schema/schema_definitions/schema_definition.yaml:249]:
```suggestion
                  FBbt:
```

Comment (nayib-jose-gloria) [reply cellxgene_schema_cli/cellxgene_schema/schema_definitions/schema_definition.yaml:249]: though if tests are passing maybe this isn't necessary....lol not worth holding this pr up but kinda interested in looking into

File: cellxgene_schema_cli/tests/test_schema_compliance.py [hunk 2]
```diff
@@ -2895,3 +2897,56 @@ def test_organism_cell_type_ontology_term_id__visium_in_tissue_0__na(self, valid
...
+                "ERROR: 'FBbt:00007294' in 'organism_tissue_ontology_term_id' is not allowed. Descendant terms of "
+                "'FBbt:00007002' are not allowed.",
+            ),
+            (
+                "na",  # Allowed for other organisms, not allowed if organism is zebrafih
+                "ERROR: 'na' in 'organism_tissue_ontology_term_id' is not a valid ontology term id of 'FBbt'.",
+            ),
+            (
+                "unkown",
+                "ERROR: 'unkown' in 'organism_tissue_ontology_term_id' is not a valid ontology term id of 'FBbt'.",
```
Comment (nayib-jose-gloria) [inline cellxgene_schema_cli/tests/test_schema_compliance.py:2935]:
```suggestion
                "unknown",
                "ERROR: 'unknown' in 'organism_tissue_ontology_term_id' is not a valid ontology term id of 'FBbt'.",
```

Comment (nayib-jose-gloria) [reply cellxgene_schema_cli/tests/test_schema_compliance.py:2935]: can you correct this for zebrafish too lol

Comment (joyceyan) [reply cellxgene_schema_cli/tests/test_schema_compliance.py]: yeah i gotchu haha

**Review by nayib-jose-gloria — APPROVED** [review]

**Pull request merged**